### PR TITLE
ENH: Add the new `pressure` generic property to `cp2k_mm`

### PR DESCRIPTION
**Pull request by BvB93**
Per the title: this PR adds a new generic property for extracting pressures over the course of a trajectory.

### CODE REVIEW

File: test/test_special_keywords.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def test_orca_init_hessian():
 
     # Test that the hessian is readable
     new_hess = parse_hessian(s.specific.orca.geom.InHessName)
-    new_hess = np.array(new_hess, dtype=np.float)
```
Comment (felipeZ) [inline test/test_special_keywords.py:206]: In Python `float`, `np.float64`, np.float` are identical. See this [deprecation paragraph](https://numpy.org/doc/stable/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated)

**Review by felipeZ — APPROVED** [review]
Looks good to me. I left a comment about floating points in Python

File: test/test_special_keywords.py (hunk 1 above)
Comment (BvB93) [reply test/test_special_keywords.py:206]: `np.float` (an alias for `builtins.float`) has recently been deprecated though, hence the change.

Comment (BvB93) [reply test/test_special_keywords.py:206]: Changing it just plain `float` (without the `np`) would also fix the issue though.

**Pull request merged**